tests/conftest.py

```diff
@@ -186,6 +186,43 @@ def multiple_credentials(tmp_path: Path) -> Path:
     return auth_file
 
 
+SENSITIVE_HEADERS = [
+    # All sensitive headers should be lower case as
+    # that's how they are presented by VCR.
+    'authorization',
+    'cookie',
+    'set-cookie',
+    'x-goog-api-key',
+    'openai-organization',
+    'openai-project',
+    'x-request-id',
+    'cf-ray',
+]
+
+
+def scrub_headers(response: dict[str, Any]) -> dict[str, Any]:
+    """Remove sensitive headers from the response.
+
+    Parameters:
+        response: The response dictionary to scrub.
+
+    Returns:
+        The scrubbed response dictionary with sensitive headers removed.
+    """
+    headers: dict[str, list[str]] = response.get('headers', {})
+    new_headers: dict[str, list[str]] = {}
+    for k, v in headers.items():
+        if k.lower() not in SENSITIVE_HEADERS:
+            new_headers[k] = v
+
+    response['headers'] = new_headers
+
+    return response
+
+
 @pytest.fixture(scope='module')
-def vcr_config():
-    return {'filter_headers': ['authorization', 'cookie', 'Set-Cookie', 'x-goog-api-key']}
+def vcr_config() -> dict[str, Any]:
+    return {
+        'filter_headers': SENSITIVE_HEADERS,
```

Comment:
Did this not work on its own? it seemed to work for `authorization`. was it just about using lowercase for `Set-Cookie`?

Reply:
It will remove authorize as it's lowercase already, but not Set-Cookie as the values it compares against are lowercase. Also required to be lowercase for the in check in scrub_headers